src/Dflydev/FigCookies/SetCookie.php

```diff
@@ -47,69 +49,69 @@ private function __construct(string $name, ?string $value = null)
         $this->value = $value;
     }
 
-    public function getName() : string
+    public function getName(): string
     {
         return $this->name;
     }
 
-    public function getValue() : ?string
+    public function getValue(): ?string
     {
         return $this->value;
     }
 
-    public function getExpires() : int
+    public function getExpires(): int
     {
         return $this->expires;
     }
 
-    public function getMaxAge() : int
+    public function getMaxAge(): int
     {
         return $this->maxAge;
     }
 
-    public function getPath() : ?string
+    public function getPath(): ?string
     {
         return $this->path;
     }
 
-    public function getDomain() : ?string
+    public function getDomain(): ?string
     {
         return $this->domain;
     }
 
-    public function getSecure() : bool
+    public function getSecure(): bool
     {
         return $this->secure;
     }
 
-    public function getHttpOnly() : bool
+    public function getHttpOnly(): bool
     {
         return $this->httpOnly;
     }
 
-    public function getSameSite() : ?SameSite
+    public function getSameSite(): ?SameSite
     {
         return $this->sameSite;
     }
 
-    public function withValue(?string $value = null) : self
+    public function withValue(?string $value = null): self
     {
-        $clone = clone($this);
+        $clone = clone $this;
 
         $clone->value = $value;
 
         return $clone;
     }
 
-    /** @param int|\DateTimeInterface|string|null $expires */
-    private function resolveExpires($expires = null) : int
+    /** @param int|DateTimeInterface|string|null $expires */
+    private function resolveExpires($expires = null): int
     {
         if ($expires === null) {
             return 0;
         }
 
         if ($expires instanceof DateTimeInterface) {
-            return $expires->getTimestamp();
+            return (int) $expires->getTimestamp();
```

Comment:
based on documentation https://www.php.net/manual/de/datetime.gettimestamp.php will always return a int

Reply:
Interesting. Why was this changed? Possibly to satisfy some static analysis? Or...?

Reply:
Before php 8, this method *could* return false on failure.